scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -185,16 +184,20 @@ const getExportModal = (
 const getHighlightsAction = (getItem: () => IArticle): IAuthoringAction => {
     return {
         label: gettext('Highlights'),
-        onTrigger: () => (
-            showModal(({closeModal}) => {
-                return (
-                    <HighlightsModal
-                        article={getItem()}
-                        closeModal={closeModal}
-                    />
-                );
-            })
-        ),
+        onTrigger: () => {
+            sdApi.highlights.fetchHighlights().then((res) => {
+                if ((res._items.length ?? 0) === 0) {
+                    ui.alert(gettext('Highlights aren\'t configured on this instance.'));
```

Comment:
Keep in mind that `an instance` doesn't mean anything for end users. For them it's THE Superdesk. They have only one, thus it'd be better to avoid such phrasing. I wouldn't also call it an instance configuration even technically same as you wouldn't call it an instance configuration if no articles have been created yet.

I'd use `No highlights have been created yet.`